### ISSUE TITLE
[데이터셋검토] MUSDB18

### ISSUE THREAD

**Issue by leemgs** [aigovsensing/dataset-review#153]
### 데이터셋 명칭

MUSDB18

### 데이터셋 저장소 URL

https://sigsep.github.io/datasets/musdb.html#musdb18-compressed-stems

### 추가 참고 사항

(Simple deThe musdb18 is a dataset of 150 full lengths music tracks (~10h duration) of different genres along with their isolated drums, bass, vocals and others stems.scription - Features, Definition...)

The musdb18 is a dataset of 150 full lengths music tracks (~10h duration) of different genres along with their isolated drums, bass, vocals and others stems.musdb18 contains two folders, a folder with a training set: "train", composed of 100 songs, and a folder with a test set: "test", composed of 50 songs. Supervised approaches should be trained on the training set and tested on both sets.All signals are stereophonic and encoded at 44.1kHz.The data from musdb18 is composed of several different sources:100 tracks are taken from the DSD100 dataset, which is itself derived from The 'Mixing Secrets' Free Multitrack Download Library (opens new window). Please refer to this original resource for any question regarding your rights on your use of the DSD100 data.46 tracks are taken from the MedleyDB (opens new window)licensed under Creative Commons (BY-NC-SA 4.0).2 tracks were kindly provided by Native Instruments originally part of their stems pack (opens new window).2 tracks are from the Canadian rock band The Easton Ellises as part of the heise stems remix competition (opens new window), licensed under Creative Commons (BY-NC-SA 3.0).n - Background, Applied tech.
